### PR TITLE
Docs: Update Ubuntu/Debian repo instructions

### PR DESCRIPTION
**Pull request by whyman**
Note that there wont be anything populated in 'stable' until we push the next tagged release

### CODE REVIEW

File: doc/install.rst [hunk 1]
```diff
@@ -28,21 +28,26 @@ Ubuntu/Mint
...
+We maintain a `Ubuntu Repository <https://pkg.gerbera.io/>`__.
 
-To install the latest tagged release (>=1.8.0):
+To install the latest tagged release (>=2.1.0):
 
 .. code-block:: sh
 
-    curl -fsSL https://gerbera.jfrog.io/artifactory/api/gpg/key/public | sudo apt-key add -
-    sudo apt-add-repository https://gerbera.jfrog.io/artifactory/debian
+    wget -O - https://pkg.gerbera.io/public.asc 2>/dev/null | gpg --dearmor - | sudo tee /usr/share/keyrings/gerbera-keyring.gpg >/dev/null
```
Comment (Martii) [inline doc/install.rst:37]: Don't know if this is "the usual thing" but still getting stdout on the binary gpg key here... it does however create it correctly in the target file. Logging might be a little messy if this is the norm.

File: doc/install.rst [hunk 2]
```diff
@@ -28,21 +28,26 @@ Ubuntu/Mint
...
+    echo "deb [signed-by=/usr/share/keyrings/gerbera-keyring.gpg] https://pkg.gerbera.io/debian/ $(lsb_release -c --short) main" | sudo tee /etc/apt/sources.list.d/gerbera.list >/dev/null
```
Comment (Martii) [inline doc/install.rst:38]: Created file normally... as [you mentioned](https://github.com/gerbera/gerbera/issues/2988#issuecomment-2048476571) the release isn't there yet:

``` console
Reading package lists... Done                                      
E: The repository 'https://pkg.gerbera.io/debian jammy Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

File: doc/install.rst [hunk 3]
```diff
@@ -28,21 +28,26 @@ Ubuntu/Mint
...
+    sudo apt-get update
```
Comment (Martii) [inline doc/install.rst:39]: as [you mentioned](https://github.com/gerbera/gerbera/issues/2988#issuecomment-2048476571) the release isn't there yet:

``` console
Reading package lists... Done                                      
E: The repository 'https://pkg.gerbera.io/debian jammy Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

File: doc/install.rst [hunk 4]
```diff
@@ -129,6 +134,27 @@ Gerbera is included in Buster_ and Sid_.
...
+    sudo apt-get install gerbera
+
+Or for the latest code install git builds:
+
+.. code-block:: sh
+
+    wget -O - https://pkg.gerbera.io/public.asc 2>/dev/null | gpg --dearmor - | sudo tee /usr/share/keyrings/gerbera-keyring.gpg >/dev/null
+    echo "deb [signed-by=/usr/share/keyrings/gerbera-keyring.gpg] https://pkg.gerbera.io/debian-git/ $(lsb_release -c --short) main" | sudo tee /etc/apt/sources.list.d/gerbera.list >/dev/null
```
Comment (Martii) [inline doc/install.rst:153]: See successful installation image at https://github.com/gerbera/gerbera/issues/2988#issuecomment-2048975661

File: doc/install.rst [hunk 5]
```diff
@@ -28,21 +28,26 @@ Ubuntu/Mint
 .. index:: Ubuntu Linux
 .. index:: Mint
 
-We maintain a `Ubuntu Repository <https://gerbera.jfrog.io/>`__.
+We maintain a `Ubuntu Repository <https://pkg.gerbera.io/>`__.
```
Comment (KarlStraussberger) [inline doc/install.rst:31]: I'm getting this friendly message for the page:
grafik

Comment (Martii) [reply doc/install.rst:31]: Yes a landing page there would be nice to have. Some sources don't though and makes finding if a release is available by directory read traversal a little harder to determine if a certain distro is supported yet.

**Comment by whyman** [comment]
Thanks for looking at this @Martii!

Will look at getting something friendlier up on when you load it directly

**Pull request merged**